telco-core/configuration/Makefile

```diff
@@ -0,0 +1,59 @@
+GOPATH  ?= $(HOME)/go
+GOBIN ?= $(GOPATH)/bin
+export HELMCONVERTOR := $(GOBIN)/helm-convert
+DOWNLOAD_URL := $(shell curl -s https://api.github.com/repos/openshift/kube-compare/releases/latest | jq -r '.assets[] | select(.name? | match("linux_amd64")) | .browser_download_url')
+ARCHIVE_NAME := kubectl-cluster_compare-linux_amd64.tar.gz
+HELM_URL := https://get.helm.sh/helm-v3.16.1-linux-amd64.tar.gz
+HELM_PKG := helm-linux-amd64.tar.gz
+
+.PHONY: check
+check: metadata_lint compare
+
+
+.PHONY: metadata_lint
+metadata_lint: kubectl-cluster_compare
+	@echo "Running kube-compare to ensure metadata.yaml is sane"
+	@COMPARE_OUTPUT=$$(./kubectl-cluster_compare -r ./reference-crs-kube-compare/metadata.yaml -f /dev/null 2>&1); \
```

Comment:
Do you really use the kubectl cluster_compare tool just to check the syntax? And not the content equivalence?

Reply:
cluster-compare tool will be used later as part of main CI once we fixed all differences between reference and source-crs.